### PR TITLE
webView: Add tweet inline image CSS styles.

### PR DESCRIPTION
**Pull request by jainkuniya**
Let me know changes :)

### CODE REVIEW

File: src/render-html/css.js [hunk 1]
```diff
@@ -277,6 +277,10 @@ ul {
   margin: 0.25em 0;
   font-size: 10px;
 }
+div.twitter-image > a > img {
```
Comment (borisyankov) [inline src/render-html/css.js]: Try to match what the web version is doing more closely.
You don't need the `div` part when it is followed by a class.
`.twitter-image img` is enough.

File: src/render-html/css.js [hunk 2]
```diff
@@ -277,6 +277,10 @@ ul {
...
+  height: auto;
+  max-width: 150px;
```
Comment (borisyankov) [inline src/render-html/css.js]: We need to restrict the height not the width. There is nothing bad about wide image.

**Comment by borisyankov** [comment]
Closing in favor of #1798

**Comment by jainkuniya** [comment]
You forgot to close it :)
Closing it 👍

**Pull request closed without merging**